src/benchmark/operators/sql_benchmark.cpp

```diff
@@ -26,9 +26,9 @@ class SQLBenchmark : public BenchmarkBasicFixture {
     // Add tables to StorageManager.
     // This is required for the translator to get the column names of a table.
     auto& storage_manager = StorageManager::get();
-    storage_manager.add_table("customer", load_table("src/test/tables/tpch/customer.tbl", 0));
-    storage_manager.add_table("lineitem", load_table("src/test/tables/tpch/lineitem.tbl", 0));
-    storage_manager.add_table("orders", load_table("src/test/tables/tpch/orders.tbl", 0));
+    storage_manager.add_table("customer", load_table("src/test/tables/tpch/minimal/customer.tbl", 0));
```

Comment:
(comment at random line)
Earlier, I missed the comment about the tests running longer now. 1m22s is definitely too long, especially if it is not distributed over thousands of tests. While it is good to have bigger tables for the tests, this is something that the CI should check. Developers should not have to wait this long unless they really want to.
Can we somehow give a parameter to the main method of google test (`./build/hyriseTest full`) that switches between the small and the bigget tableset?

Reply:
There is no such thing as a small table set.
- The one included in src/test/tables/tpch/ is what dbgen generates with a scale factor of 0.001. Choosing a lower scale factor (0.0009, e.g.) will drop to incredibly small tables immediately (6 lineitems, e.g.) that I would consider not big enough for testing.
- The three tables in src/test/tables/tpch/minimal are "legacy" tables once manually created and used in some non-tpch tests.

As to runtime, I agree that it is too long. Will try to investigate whether manual pruning of the tables generated at scale factor 0.001 is feasible

Reply:
I have manually reduced the size of the lineitem table and now the tests finish in 21s. Is this acceptable for the time being?

Reply:
I made a change to abstract_ast_node.cpp and timed the compilation plus test execution on master. That took me 30s. This would almost double the time needed for an iteration.

Reply:
alright, separate hyriseTestTPCH executable then? I think we really need some tests that operate on big tables.

Reply:
Absolutely agreed. We don't want a DB that only works with 10 rows.

Maybe a `hyriseTestExpensive`? That way we could also move things like `OperatorsTableScanTest.ScanOnWideDictionaryColumn (871 ms)` into that binary.

Reply:
rename `hyriseTest` to `hyriseTestBasic/Core/SomethingLikeThat` as well?

Reply:
Mmh, not necessary if you ask me. I'm already unhappy enough that we can't do `make test`. But if you think that it's more consistent, we can do that.

Reply:
Is there any way to have regular and expensive tests in the same file? Some `TEST_F` magic?

Reply:
Yes, there is: https://stackoverflow.com/a/26461698/2204581

Reply:
Yes, there is: https://stackoverflow.com/a/26461698/2204581
This way, we don't even need a separate binary but could just it as an argument: `./build/hyriseTest thorough`. Or `full`, because that's easier to remember.

Reply:
you think doing this via args is the best way? Because argc and argv also get passed to gtest and would probably need to be re-processed if we want to add custom commands. I'd prefer two different main()/binaries linking against the same hyriseTestLib

Reply:
Ah, I forgot that googletest does its own things with the args. Yes, in that case, two binaries are better.